system-test/data/read-rows-retry-test.json

```diff
@@ -326,8 +326,266 @@
         [ "y" ],
         [ "z" ]
       ]
-    }
+    },
 
 
```

Comment:
A test is run for each json object in this file. The test starts a mock server and sends an initial request. It has a predetermined list of responses to send back to the client which are handled by the retry logic. The mock server collects a list of all the requests that were made and checks to see that they are all correct.

Reply:
In practice I think it just performs a simple read. There was a bug where it used to do a full table scan, but a change was made so now it just adds `"rowKeys": [],"rowRanges": [{}]` to do a simple read. For these tests the responses are predetermined so this has no impact on the responses in the test.